patches/chrome-browser-resources-settings-clear_browsing_data_dialog-clear_browsing_data_dialog.html.patch

```diff
@@ -0,0 +1,99 @@
+diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html
+index 1a086904cc3c93cb0ff6e0ad0eab0de850638591..dfa28f86fdf53fe1f6a319f9a8129f022b1d01b5 100644
+--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html
++++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.html
+@@ -117,6 +117,7 @@
+             selected="{{prefs.browser.last_clear_browsing_data_tab.value}}">
+           <paper-tab id="basicTabTitle">$i18n{basicPageTitle}</paper-tab>
+           <paper-tab id="advancedTabTitle">$i18n{advancedPageTitle}</paper-tab>
++          <paper-tab id="onExitTabTitle">$i18n{onExitPageTitle}</paper-tab>
+         </paper-tabs>
+       </div>
+       <div slot="body">
+@@ -242,6 +243,68 @@
+                 disabled="[[clearingInProgress_]]">
+             </settings-checkbox>
+           </div>
++          <div id="on-exit-tab">
```

Comment:
this is way too much to patch. This needs to be inserted using the template system so the bulk of the code goes in another file

Reply:
@bridiver do you have an example of the template system?

Reply:
Maybe you can check https://github.com/brave/brave-core/pull/1218/files#diff-9c349bb8a82ed234202615818b9f63fbR32 and https://github.com/brave/brave-core/pull/1218/files#diff-729401eebc8530bbf9177e90aae8b2e4.
Here we wrap all our stuff into one component and just patched two lines and passed what we need (prefs) into it.